docs/brokers-journalspecs.rst

```diff
@@ -44,9 +44,11 @@ Etcd Revisions
 JournalSpecs retrieved by the ``gazctl`` tool will include their respective
 Etcd modification revisions as field ``revision`` within the rendered YAML.
 
-When applying YAML specs via ``gazctl journals apply``, any specs which omit
-``revision`` assume a value of zero (implying the journal must not exist).
-The revisions of specs are always compared to the current Etcd store revision,
+When applying YAML specs via ``gazctl journals apply`` the only allowed values of
+``revision`` are 0 and -1, and if specified must only exist on the leaf-level journals
```

Comment:
This is stricter than the actual implementation (0 is the empty value and is allowed on directory nodes), but since we are not going to push down revision, this seemed like a simple way to state the intent